docs/rules/no-else-return.md

```diff
@@ -88,9 +88,10 @@ function foo() {
 
 function foo() {
     if (x) {
-        if (z) {
+        if (y) {
             return y;
         }
+        return x;
```

Comment:
I don't think this change is correct. The addition would cause the rule to error, while this is an example of correct code.